kong.conf.default

```diff
@@ -1809,3 +1809,7 @@ G# -----------------------
                                   # behavior and explicitly instructs Kong which
                                   # OpenResty installation to use.
 
+#kong_node_id =                   # Node ID for the Kong node. Every Kong node
```

Comment:
```suggestion
#node_id =                   # Node ID for the Kong node. Every Kong node
```

Reply:
`node_id` is better. `KONG_KONG_NODE_ID` in environment variable will looks strange.

Reply:
This was an oversight, fixed in the latest push.